CONTRIBUTING.rst

```diff
@@ -146,21 +156,11 @@ Running System Tests
     `docs <https://cloud.google.com/storage/docs/authentication#generating-a-private-key>`__
     for more details.
 
-- Once you have downloaded your json keys, set the environment variable
+- Once you have downloaded your json keys, set the environment variable 
   ``GOOGLE_APPLICATION_CREDENTIALS`` to the absolute path of the json file::
 
    $ export GOOGLE_APPLICATION_CREDENTIALS="/Users/<your_username>/path/to/app_credentials.json"
 
-**************************
```

Comment:
If I'm following the plot correctly, it looks like it got dumped because it's [no longer here](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/CONTRIBUTING.rst). But unexpectedly, I couldn't find the commit that dropped it.